### ISSUE TITLE
include idiomatic nomenclature to elemTypeGeometry

### ISSUE THREAD

**Issue by mvanzulli** [ONSAS/ONSAS.m#326]
Maye we could change this nomenclature input:

elements(1).elemTypeGeometry = [[2 sqrt(Ac) sqrt(Ac)] ; to elements(1).elemTypeGeometry = ['square' sqrt(Ac) sqrt(Ac)] ;

**Comment by jorgepz** [comment]
good idea... however i think it would be better something like
```
elements(1).elemTypeGeometry = { 'rectangular' sqrt(Ac) sqrt(Ac) } ;
```
since in octave if you want to mix strings and floats, a cell would be better, than a vector

**Comment by mvanzulli** [comment]
Closed via #414

**Comment by jorgepz** [comment]
reopen since some examples were not working after this
